contrib/dockerd-rootless.sh

```diff
@@ -13,6 +10,9 @@
 #
 # See the documentation for the further information.
 
+# The default flags used for running dockerd
+: ${DOCKERD_ROOTLESS_DOCKERD_FLAGS:="--experimental --exec-opt native.cgroupdriver=none --exec-opt native.restrict_oom_score_adj=1 --userland-proxy --userland-proxy-path=$(which rootlesskit-docker-proxy)"}
```

Comment:
@seemethere @andrewhsu (?)

Please update docker-rootless-extras.tgz to include rootlesskit-docker-proxy before merging this PR.

Reply:
merged in https://github.com/docker/docker-install/pull/109